modules/library/sample-data/src/main/java/org/geotools/test/OnlineTestCase.java

```diff
@@ -243,7 +244,13 @@ protected final void setUp() throws Exception {
                 // disable the test
                 fixture = null;
                 // leave some trace of the swallowed exception
-                java.util.logging.Logger.getGlobal().log(java.util.logging.Level.INFO, "", e);
+                java.util.logging.Logger.getGlobal()
```

Comment:
While you're at it, can you add imports and avoid fully qualified class references?
I've guessing it would avoid most of the line wrapping here.

Reply:
Done